es/lecciones/corpus-paralelo-lfaligner.md

```diff
@@ -0,0 +1,278 @@
+
+{% include figure.html filename="lfaligner-16.jpg" caption="Al presional *split*, obtendremos este resultado" %}
+
+Como todavía hay una porción de texto que debe colocarse en la celda siguiente, repetimos el procedimiento.
+
+{% include figure.html filename="lfaligner-17.jpg" caption="Presionamos nuevamente *split* al comienzo del fragmento que queremos desplazar" %}
+
+Gracias a nuestra edición, el texto de los segmentos 10, 11 y 12 ha quedado perfectamente alineado.
+
+## Cierre del programa: Windows
+Cuando termines de revisar el documento, escoge la opción *save & exit* (guardar y salir) en el menú *file* (archivo).
+
+{% include figure.html filename="lfaligner-18.jpg" %}
+
+Tras hacer esto, la herramienta de edición se cerrará. Regresa a la ventana principal del programa para cerrarlo completamente.
+
+{% include figure.html filename="lfaligner-19.jpg" caption="El programa nos da la opción de generar un archivo de memoria de traducción" %}
+
+LF Aligner ofrece la opción de exportar nuestro documento con formato de memoria de traducción (en este caso tmx). Este tipo de archivos sirven exclusivamente para alimentar software de traducción asistida, ya sea para creación de bases terminológicas personalizadas o para apoyo en las tareas de traducción asistida como traducción automática de segmentos. Para efectos de este tutorial, no es necesario hacer esto. Escoge la opción *no* y presiona *next* (siguiente) para finalizar. Aparecerá una última ventana avisándonos que el programa se ha cerrado exitosamente.
+
+**Cierre del programa en Linux y Mac**
+Justo después de crear el archivo xls, el cual se abrirá inmediatamente, el programa preguntará si deseamos crear un archivo de memoria de traducción. Seleccionamos "no", introduciendo *n* y presionando *entrar*. El software mostrará un último mensaje, indicándonos que el proceso ha finalizado. Basta con presionar *entrar* una vez más para terminar el programa. Luego de esto podemos cerrar la terminal sin ningún problema.
+
+## Ubicación del documento alineado
+Si seguiste las indicaciones anteriores sobre el nombramiento y almacenamiento de los textos, te será muy fácil encontrar el documento. Abre la carpeta en cuestión. Ahí verás una nueva carpeta cuyo nombre comienza con la palabra *align*. Dentro de ella encontrarás los documentos individuales en formato txt que corresponden al texto segmentado por el software y un archivo de planilla de datos (formato xls) que contiene el texto alineado y editado por nosotros.
+
+{% include figure.html filename="lfaligner-20.jpg" caption="El archivo exportado en formato xls aparecerá en la carpeta correspondiente" %}
+
+También puedes [**descargar**](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/assets/corpus-paralelo-lfaligner/align_2018.08.03_14.09.46/DDHH_es-es-en-fr-pt.xls) el documento alineado y explorarlo para aprender más sobre el funcionamiento de este software.
```

Comment:
Please turn this into a relative link that points to `/assets/corpus-paralelo-lfaligner/...`